src/modules/wallet/utils/hwManager.js

```diff
@@ -22,7 +22,7 @@ const getAccountBundle = async (deviceId, network, offset) => {
     publicKeyList.push(publicKey);
```

Comment:
I guess this file is just formatting?

Reply:
@oskarleonard Yes it is. Just one line was removed and the rest are formatting changes

Reply:
please revert all the changes on this file

Reply:
@soroushm I had to remove a line or two of dead code and the rest is formatting changes. So why do we need to revert all changes on the file?

Reply:
we dont want to toch any existing utils, create the utlist you need on 'src/modules/hardwareWallet/utils"

Reply:
Fixed